app/components/ConfirmModal.js

```diff
@@ -0,0 +1,34 @@
+import Modal from "react-modal";
+
+const ConfirmModal = ({
+  appElement = "#root",
+  className,
+  isOpen,
+  toggleModal,
+  title,
+  content,
+}) => (
+  <Modal
+    appElement={appElement ? appElement : Modal.setAppElement("#root")}
```

Comment:
Does it make sense to have this inside the component? https://reactcommunity.org/react-modal/accessibility/#app-element

Reply:
Good call - this was a holdover from DIMES where there is a `#root` element.

Reply:
We do need some kind of `#root` element because it's how react-modal adds aria-hidden="true" to everything else on the page when the modal is open. I think we need to add a `<div id="root">` wrapper to layout.js just inside the body to use the modal correctly. I just wasn't sure `Modal.setAppElement()` needed to be called inside the component like this. I guess it is giving us the flexibility to use other DOM elements as the AppElement to hide things other than the standard `#root` that should be set?

Reply:
Yes, exactly. The modal will work as long as you pass it an element (as in the tests) so this gives us the flexibility to use whatever element we want.